introduction.md

```diff
@@ -0,0 +1 @@
+Hi I would tell you my name, but I think you already know what it is.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.